sys-apps/ndctl/ndctl-64.1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1 linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="test"
+
+RDEPEND="dev-libs/json-c
+	sys-apps/keyutils
+	sys-apps/kmod
+	virtual/udev
+	sys-apps/util-linux"
```

Comment:
Please sort this too :)

Reply:
I had actually sorted it the first time but i sorted by package name rather than by category.  Now its sorted by category then package name.